auth/store.go

```diff
@@ -1279,3 +1314,17 @@ func (as *authStore) HasRole(user, role string) bool {
 func (as *authStore) BcryptCost() int {
 	return as.bcryptCost
 }
+
+func (as *authStore) saveConsistentIndex(tx backend.BatchTx) {
+	if as.syncConsistentIndex != nil {
```

Comment:
Do we want to add a warning here if it is nil? IIUC, if this is nil, we lose correctness of auth store after a store restart.

Reply:
i have added a error log. thanks. PTAL.